vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java

```diff
@@ -400,17 +400,44 @@ else if (Y_POSITION.equals(key)) {
    */
   @Override
   public String getConfigureName() {
+    final StringBuilder sb = new StringBuilder("");
+    boolean any = false;
```

Comment:
Is `any` just for tracking if `sb` is non-empty? If so, you could just call `!sb.isEmpty()`.

Reply:
There's no `isEmpty()` method for StringBuilder, but I can use `sb.length() > 0`